client/mapreduce/src/main/java/org/apache/accumulo/core/client/mapreduce/lib/impl/ConfiguratorBase.java

```diff
@@ -337,7 +337,8 @@ public static void setZooKeeperInstance(Class<?> implementingClass, Configuratio
   @Deprecated
   public static org.apache.accumulo.core.client.Instance getInstance(Class<?> implementingClass,
       Configuration conf) {
-    return Connector.from(getClient(implementingClass, conf)).getInstance();
+    return org.apache.accumulo.core.client.Connector.from(getClient(implementingClass, conf))
+        .getInstance();
```

Comment:
Any other reason to use fully qualified other than remove the import?  Could this change make it more difficult for tools (maven plugins, IDEs, etc) to find uses of Connector?

Reply:
The only change this fixes is a workaround for https://bugs.java.com/view_bug.do?bug_id=8032211 which isn't fixed until Java 9.